README.md

```diff
@@ -58,144 +59,118 @@ The tomcat module requires [puppetlabs-stdlib](https://forge.puppetlabs.com/pupp
 puppet module upgrade puppetlabs-stdlib
 ~~~
 
-###Beginning with tomcat
+### Beginning with tomcat
 
-The simplest way to get Tomcat up and running with the tomcat module is to install the Tomcat package from EPEL...
+The simplest way to get Tomcat up and running with the tomcat module is to install the Tomcat source and start the service:
 
-~~~
-class { 'tomcat':
-  install_from_source => false,
+```puppet
```

Comment:
The tomcat module can install arbitrary numbers of software copies, so a single class is not very useful. I updated the design to allow the base class to set defaults, though it doesn't do anything by itself (and before the redesign, was similar).

I did update the docs to clarify this though.